openqa-label-known-issues

```diff
@@ -27,6 +24,23 @@ OPENQA_CLI_RETRY_SLEEP_TIME_S=${OPENQA_CLI_RETRY_SLEEP_TIME_S:-120}
 MOJO_CONNECT_TIMEOUT=${MOJO_CONNECT_TIMEOUT:-30}
 client_args=(api --header 'User-Agent: openqa-label-known-issues (https://github.com/os-autoinst/scripts)' --host "$host_url" --retries="$retries")
 
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS] OPENQA_JOB_URL
+
+Takes an openQA job URL, looks for matching "known issues", for example from
+progress.opensuse.org, labels the job and retriggers if specified in the issue
+(see the source code for details how to mark tickets).
```

Comment:
Please add some text on what a known issues is and how it is found in progress.
Also maybe add an example on these labels. I'm not even sure if this refers to labels on openQA jobs or on some magic strings in progress ticket subjects.
Looking at the source code I cannot answer this question - it would be more of a prerequisite to even understand the source.